test/github_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule ProtectTest.Github do
+  use ExUnit.Case
+  alias Protect
```

Comment:
What does this line do?

Reply:
@shouston3 you're right, it does not look like the `alias Protect` is _used_ in this file. 🤔

Reply:
Good spot, I'm not using Protect in this file, and even if I was, aliasing it as itself is completely unnecessary (which I'm also doing here: https://github.com/dwyl/protect-master-branch/blob/6dafcabbbad1492e096590ef406a1c3987fc393c/test/protect_test.exs#L4).
I'l remove these two unused aliases

Reply:
@Danwhy would "dogma" have picked this up? e.g:
https://github.com/dwyl/hits-elixir/blob/e6a82aa3b1d36346a9fb47eb91a3ca2ffafe2d9e/package.json#L11-L14
(_not required for `this` project but useful for removing un-used code_)

Reply:
Yeah it probably would have. We've also used "credo" in the past. I'll raise an issue around adding a linter in case we do more work on this in the future.